README.md

```diff
@@ -24,6 +24,13 @@ For convenience, you can alternatively run the `make.sh` script.
 - CMake for generating the build files
 - Python 3 for the [code generator](https://github.com/toru/h2olog/blob/v2/misc/gen-quic-bpf.py)
 - [BCC](https://iovisor.github.io/bcc/) (>= 0.11.0) [installed](https://github.com/iovisor/bcc/blob/master/INSTALL.md) on your system
+- systemtap for Linux to enable DTrace for H2O
+
+If you use Ubuntu 20.04 or later, you can install dependencies with `apt`:
```

Comment:
I think we can cut down the sentence to: "For Ubuntu 20.04 or later, you can install dependencies with:" -- `apt` is stripped because it's kind of redundant given the example command.